app/components/emoji/emoji.js

```diff
@@ -11,7 +11,7 @@ import {
 } from 'react-native';
 
 import CustomPropTypes from 'app/constants/custom_prop_types';
-import ImageCacheManager from 'app/utils/image_cache_manager';
```

Comment:
I still want to manage the cache ourselves as when you logout we clear the folder, the only other way is to be able to tell fastImage where to store the cache